composer.json

```diff
@@ -1,43 +1,44 @@
 {
-  "name": "silverstripe/silverstripe-search-service",
-  "description": "Elastic App Search Functionality",
-  "type": "silverstripe-vendormodule",
-  "license": "BSD-3-Clause",
-  "authors": [
-	{
-      		"name": "Will Rossiter",
-      		"email": "will@fullscreen.io"
-  	},
-	{
-		"name": "Aaron Carlino",
-		"email": "aaron@silverstripe.com"
-	}
-  ],
-  "require": {
-      "symbiote/silverstripe-queuedjobs": "^4.0.0",
-      "elastic/app-search": "^7.6",
-      "silverstripe/versioned": "^1"
-  },
-  "require-dev": {
-      "phpunit/phpunit": "^5.7",
-      "squizlabs/php_codesniffer": "^3"
-  },
-  "scripts": {
-    "lint": "phpcs --extensions=php src/ tests/",
-    "syntax-check": "find src/ tests/ -type f -name '*.php' -exec php -l {} \\;",
-    "lint-clean": "phpcbf src/ tests/"
-  },
-  "extra": {
-      "branch-alias": {
-          "dev-master": "1.x-dev"
-      }
-  },
-  "autoload": {
-      "psr-4": {
-          "SilverStripe\\SearchService\\": "src/",
-          "SilverStripe\\SearchService\\Tests\\": "tests/"
-      }
-  },
-  "prefer-stable": true,
-  "minimum-stability": "dev"
+    "name": "silverstripe/silverstripe-search-service",
+    "description": "Elastic App Search Functionality",
+    "type": "silverstripe-vendormodule",
+    "license": "BSD-3-Clause",
+    "authors": [
+        {
+            "name": "Will Rossiter",
+            "email": "will@fullscreen.io"
+        },
+        {
+            "name": "Aaron Carlino",
+            "email": "aaron@silverstripe.com"
+        }
+    ],
+    "require": {
+        "php": "^7.4 || ^8.0",
+        "symbiote/silverstripe-queuedjobs": "^4.0.0",
+        "silverstripe-terraformers/app-search-php-8": "^7.14.0",
+        "silverstripe/versioned": "^1"
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^9.5",
+        "squizlabs/php_codesniffer": "^3"
+    },
+    "scripts": {
+        "lint": "phpcs --extensions=php src/ tests/",
+        "syntax-check": "find src/ tests/ -type f -name '*.php' -exec php -l {} \\;",
+        "lint-clean": "phpcbf src/ tests/"
+    },
+    "extra": {
+        "branch-alias": {
+            "dev-master": "1.x-dev"
+        }
+    },
```

Comment:
```suggestion
```


I don't like branch aliases. I've created a new `1` and `1.0` branches from master and I've changed the default branch to `1` - could you please update this PR to target 1.

We'll keep the master branch there for now so that existing projects can use that, though you should make a follow up task to update any projects requirement from `dev-master` to `^1`

Reply:
We'll also need a follow up PR to remove the branch-alias from the `master` branch